.circleci/config.yml

```diff
@@ -122,7 +122,7 @@ jobs:
           command: |
               . venv/bin/activate && pip install --no-cache-dir --upgrade -e . --progress-bar off && mkdir packages
               cd dash-renderer && renderer build && python setup.py sdist && mv dist/* ../packages/ && cd ..
-              git clone --depth 1 https://github.com/plotly/dash-core-components.git
+              git clone --depth 1 -b import-test https://github.com/plotly/dash-core-components.git
```

Comment:
Remove this branch after merging https://github.com/plotly/dash-core-components/pull/903